apps/site/package.json

```diff
@@ -3,8 +3,9 @@
   "type": "module",
   "scripts": {
     "scripts:release-post": "cross-env NODE_NO_WARNINGS=1 node scripts/release-post/index.mjs",
-    "dev": "cross-env NODE_NO_WARNINGS=1 next dev --turbopack",
-    "serve": "npm run dev",
+    "dev": "npm run build-blog-data -- --watch & cross-env NODE_NO_WARNINGS=1 next dev --turbopack",
```

Comment:
Please use `node --watch` instead of Chokidar.

Reply:
chokidar removed, it seems like `--watch` works just fine... not sure what I messed up before 😕 

sorry for the silly mistake 😓